packages/app-project/src/components/AuthModal/shared/components/withFormik/withFormik.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Formik } from 'formik'
+
+function withFormik (WrappedComponent) {
+  class HOC extends React.Component {
+    constructor () {
+      super()
+
+      this.handleBlurWithCallback = this.handleBlurWithCallback.bind(this)
+      this.handleChangeWithCallback = this.handleChangeWithCallback.bind(this)
+    }
+
+    handleBlurWithCallback (event, handleBlur, formikProps) {
+      handleBlur(event)
+      this.props.onBlur(event, formikProps)
+    }
+
+    handleChangeWithCallback (event, handleChange, formikProps) {
+      handleChange(event)
+      this.props.onChange(event, formikProps)
+    }
+
+    render () {
+      const { initialValues, onSubmit } = this.props
+      return (
+        <Formik
+          initialValues={initialValues}
+          onSubmit={onSubmit}
+          {...this.props}
```

Comment:
I can never remember how the spread and rest operators work, but are you passing `initialValues` and `onSubmit` twice here, since they are included in `this.props`?

Reply:
Checking the React dev tools I only see each expected prop once.